<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondaryWindow</class>
 <widget class="QWidget" name="SecondaryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EE Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="iidExpButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>IID 
Exponent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ccExpButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost
 Constrained
 Exponent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ccompExpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>Constant
 Composition
Exponent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
